src/Symfony/Component/Security/Http/RememberMe/PersistentRememberMeHandler.php

```diff
@@ -86,18 +84,28 @@ public function processRememberMe(RememberMeDetails $rememberMeDetails, UserInte
             throw new AuthenticationException('The cookie has expired.');
         }
 
+        return parent::consumeRememberMeCookie($rememberMeDetails->withValue($persistentToken->getLastUsed()->getTimestamp().':'.$rememberMeDetails->getValue().':'.$persistentToken->getClass()));
+    }
+
+    protected function processRememberMe(RememberMeDetails $rememberMeDetails, UserInterface $user): void
```

Comment:
Turning a public method into a protected one is a BC break. That cannot be done in a patch version.